catroidTest/src/org/catrobat/catroid/test/io/StorageHandlerTest.java

```diff
@@ -206,29 +204,21 @@ public void testDefaultProject() throws IOException {
 		File testFile = new File(imagePath);
 		assertTrue("Image " + backgroundLookList.get(0).getLookFileName() + " does not exist", testFile.exists());
 
-		// Test the 4 moles
-		for (int i = 0; i < 4; i++) {
-			assertEquals("not the right number of bricks in the first script", 12, projectManager.getCurrentProject()
-					.getSpriteList().get(i + 1).getScript(0).getBrickList().size());
-			assertEquals("not the right number of bricks in the second script", 4, projectManager.getCurrentProject()
-					.getSpriteList().get(i + 1).getScript(1).getBrickList().size());
+		assertEquals("not the right number of bricks in the first script", 3, projectManager.getCurrentProject()
+				.getSpriteList().get(1).getScript(0).getBrickList().size());
+		assertEquals("not the right number of bricks in the second script", 4, projectManager.getCurrentProject()
+				.getSpriteList().get(1).getScript(1).getBrickList().size());
 
-			//test if images are existing:
-			List<LookData> catroidLookList = currentProject.getSpriteList().get(i + 1).getLookDataList();
-			assertEquals("wrong number of pictures in catroid sprite", 3, catroidLookList.size());
+		ArrayList<LookData> catroidLookList = currentProject.getSpriteList().get(1).getLookDataList();
```

Comment:
This is the reason why Jenkins tests(ed) the merge of the pull request branch and the develop branch.

Since I'm currently having problems with Android Studio, I won't solve it myself. :cat: